src/components/Person/Person.jsx

```diff
@@ -1 +1,43 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const whicSex = sex === 'm'
+    ? `wife`
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+
+      <p className="Person__partner">
+        {isMarried ? `${partnerName} is my ${whicSex}`
+          : 'I am not married'}
```

Comment:
```suggestion
        {isMarried 
          ? `${partnerName} is my ${whicSex}`
          : 'I am not married'}
```
It`s better to put each expression of the ternary operator at a new line

Reply:
ok